index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Cual es tu nombre?')
```

Comment:
Acá faltó punto y coma despues de declarar la constante.
```suggestion
const nombreUsuario = prompt('Cual es tu nombre?');
```